### ISSUE TITLE
`code:: console` is illegible

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/sphinx_rtd_theme#787]
Using Sphinx 2.x and the following:

```rst
.. code:: console

    /bin/sh -c 'echo hello'

```

The output doesn't meet a11y guidelines around type contrast

![image](https://user-images.githubusercontent.com/1140183/61559791-c0680600-aa27-11e9-851b-6d6bde4f62a5.png)

**Comment by jessetan** [comment]
This is caused by the missing shell prompt, so the line is lexed as output.
If the line is preceded by `$` or `#`, it has more readable contrast:

<img width="203" alt="Screen Shot 2019-07-20 at 16 30 11" src="https://user-images.githubusercontent.com/3284111/61579915-cd5e2580-ab0b-11e9-91e7-8ea763ff9117.png">

**Comment by agjohnson** [comment]
Ah, i see. I wasn't seeing this formatting in other builds however, it was rendered with a darker font. This might have been an issue of local development, I'll try to confirm.

**Comment by agjohnson** [comment]
We can address our own docs with #794, this issue can explore whether we should alter the colors for console code STDOUT to have more contrast

**Comment by agjohnson** [comment]
More specifically, comment blocks on the code blocks is not legible because there is very little contrast.
